### PR TITLE
MySQL 8.0 support

### PR DESCRIPTION
**Pull request by 200MPH**
ENCRYPT() function is depricated in MySQL 8
I have added additional notes and statement when using MySQL 8.

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit 91d98f9ec268d582fd1902e3be67f2273cd54b19

https://deploy-preview-2883--nostalgic-ptolemy-b01ab8.netlify.com

File: docs/email/postfix/email-with-postfix-dovecot-and-mysql/index.md [hunk 1]
```diff
@@ -171,15 +171,24 @@ Now that the database and tables have been created, add some data to MySQL.
...
-1.  Add email addresses to the `virtual_users` table. The `domain_id` value references the `virtual_domain` table's `id` value. Replace the email address values with the addresses that you wish to configure on the mailserver. Replace the `password` values with strong passwords.
+2.  Add email addresses to the `virtual_users` table. The `domain_id` value references the `virtual_domain` table's `id` value. Replace the email address values with the addresses that you wish to configure on the mailserver. Replace the `password` values with strong passwords.
 
         INSERT INTO `mailserver`.`virtual_users`
           (`id`, `domain_id`, `password` , `email`)
         VALUES
           ('1', '1', ENCRYPT('password', CONCAT('$6$', SUBSTRING(SHA(RAND()), -16))), 'email1@example.com'),
           ('2', '1', ENCRYPT('password', CONCAT('$6$', SUBSTRING(SHA(RAND()), -16))), 'email2@example.com');
+          
+{{< note >}}NOTE! ENCRYPT() function is depricated in MySQL 8.0!{{< /note >}}
```
Comment (bagasme) [inline docs/email/postfix/email-with-postfix-dovecot-and-mysql/index.md]: Typo. It should be `deprecated`

File: docs/email/postfix/email-with-postfix-dovecot-and-mysql/index.md [hunk 2]
```diff
@@ -171,15 +171,24 @@ Now that the database and tables have been created, add some data to MySQL.
...
+If you are using MySQL >= 8.0 you must use below SQL statment:
```
Comment (bagasme) [inline docs/email/postfix/email-with-postfix-dovecot-and-mysql/index.md]: Typo again. It should be `statement`

**Comment by wildmanonline** [comment]
Thanks for the PR @200MPH. Since this is a bit old now (sorry for the delay!), some of these changes have already been made in this guide. For instance, since `ENCRYPT()` has been deprecated in newer MySQL versions, a new method of generating a hashed password has been implemented. We are able to include the typo fix that you made. Thanks for that!

**Review by wildmanonline — APPROVED** [review]

**Review by nmelehan — APPROVED** [review]

**Pull request merged**